src/test/peerfinder/PeerFinder_test.cpp

```diff
@@ -24,6 +24,7 @@
 #include <ripple/peerfinder/impl/Logic.h>
 #include <ripple/protocol/PublicKey.h>
 #include <ripple/protocol/SecretKey.h>
+#include "ripple/beast/unit_test/suite.hpp"
```

Comment:
Good spotting! Sorry I missed that.

Reply:
Clangd is constantly trying to sneak these in on me. I checked and sadly there is no clang-format setting that can catch this for us.

Reply:
Thanks @thejohnfreeman , fixed !

I think that `clangd` is correct, see how the source directory is added in generated build files as a `-I` parameter, rather than `-isystem`. But that's a discussion for some other time and place.

Reply:
I think the opinions around this, held by people and tools, are conflicting:

- There is a convention that includes of system headers should use angles and non-system headers should use quotes.
- Some IDEs treat system headers differently.
- Some compilers turn off warnings for system headers.

The first convention turns into a problem for dependent packages. Those packages want to treat libxrpl headers as system headers. If our headers include each other using quotes, then they are violating the convention. There is an alternative convention, which we follow, to include all public headers with angles, and who is to say which convention is right? It's just preference.

clangd should just make this configurable, but [progress](https://github.com/clangd/clangd/issues/1247) is slow.